lib/linguist/languages.yml

```
@@ -8362,6 +8362,19 @@ hoon:
   extensions:
   - ".hoon"
   language_id: 560883276
+iCalendar:
+  type: data
+  color: "#ec564c"
+  extensions:
+  - ".ics"
+  - ".ical"
+  tm_scope: source.iCalendar
+  aliases:
+  - iCal
+  ace_mode: text
```

Properties files are [supported](https://mkslanc.github.io/ace-playground/?value=Ly9zYW1wbGVzL2hlbGxvIHdvcmxkCgovLyBFYXNpZXN0IHdheSB0byBydW4gQUNFIGVkaXRvciBpbiBjb250YWluZXIgd2l0aCBpZD0iZXhhbXBsZSIKYWNlLmVkaXQoImV4YW1wbGUiLCB7CiAgICB0aGVtZTogImFjZS90aGVtZS9naXRodWJfZGFyayIsCiAgICBtb2RlOiAiYWNlL21vZGUvcHJvcGVydGllcyIsCiAgICB2YWx1ZTogIkJFR0lOOlZDQUxFTkRBUlxuVkVSU0lPTjoyLjBcbk1FVEhPRDpQVUJMSVNIXG5DTEFTUzpQVUJMSUMiCn0pO1wwXDA8c2NyaXB0IHNyYz0iaHR0cHM6Ly93d3cudW5wa2cuY29tL2FjZS1idWlsZHNAbGF0ZXN0L3NyYy1ub2NvbmZsaWN0L2FjZS5qcyI%2BPC9zY3JpcHQ%2BCjxkaXYgaWQ9J2V4YW1wbGUnIHN0eWxlPSJoZWlnaHQ6IDEwMCUiPjwvZGl2Pg%3D%3D#hello-world) in ACE too.

```suggestion
  ace_mode: properties
```

---

Good to know, thanks!